### PR TITLE
Feature/frontend loggings

### PR DESCRIPTION
**Pull request by achen2401**
address https://www.pivotaltracker.com/story/show/179599856
Add frontend loggings for:

- MME Calcuation results (both failed and calcuated)
- Click event for education material links
- flag/alerts

### CODE REVIEW

**Review by ivan-c — APPROVED** [review]

File: src/components/Landing.js [hunk 1]
```diff
@@ -110,6 +116,8 @@ export default class Landing extends Component {
     if (!message) return;
     if (!level) level = "info";
     if (!params) params = {};
+    if (!params.tags) params.tags = [];
+    params.tags.push("cosri-frontend");
```
Comment (pbugni) [inline src/components/Landing.js:120]: perfect, thank you!

File: src/components/Landing.js [hunk 2]
```diff
@@ -91,7 +91,13 @@ export default class Landing extends Component {
...
         //look for medication that contains NDC code but not RxNorm Code, or contains all necessary information (NDC Code, RxNorm Code and Drug Class) but no MME
         if (isOpioid && item["NDC_Code"] && (!item["RXNorm_Code"] || !item["MME"])) {
-          errors.push(`Medication, ${item["Name"]}, did not have an MME value returned, total MME and the MME overview graph are not reflective of total MME for this patient.`)
+          errors.push(`Medication, ${item["Name"]}, did not have an MME value returned, total MME and the MME overview graph are not reflective of total MME for this patient.`);
+          //log failed MME calculation
+          this.writeToLog(`MME calculation failure: Name: ${item.Name} NDC: ${item.NDC_Code}`, "error");
+        }
+        if (item.MME) {
+          //log MME calculated if present
+          this.writeToLog(`MME Calculated: Name: ${item.Name} NDC: ${item.NDC_Code} RxNorm: ${item.RXNorm_Code} MME: ${item.MME}`);
```
Comment (pbugni) [inline src/components/Landing.js:100]: please add `tags=['mme-calc']` or the like here

File: src/components/Landing.js [hunk 3]
```diff
@@ -200,6 +208,15 @@ export default class Landing extends Component {
       }
     }
   }
+  initEvents() {
+    //education material links
+    document.querySelectorAll(".education").forEach(item => {
+      item.addEventListener("click", (e) => {
+        this.writeToLog(`Education material: ${e.target.title}`);
```
Comment (pbugni) [inline src/components/Landing.js:215]: please add `tags=['education']` or the like here

File: src/components/Landing.js [hunk 4]
```diff
@@ -386,6 +399,8 @@ export default class Landing extends Component {
                       text: subitem.flagText + (subitem.flagDateText ? ` (${datishFormat('',subitem.flagDateText)})`: ""),
                       priority: subitem.priority || 100
                     });
+                    //log alert
+                    this.writeToLog("alert flag: "+subitem.flagText, "warn");
```
Comment (pbugni) [inline src/components/Landing.js:403]: please add `tags=['alert']` or the like, here

**Review by pbugni — CHANGES_REQUESTED** [review]

**Review by pbugni — APPROVED** [review]

**Pull request merged**